sdk/eventhub/event-processor-host/src/modelTypes.ts

```diff
@@ -116,12 +116,16 @@ export interface FromTokenProviderOptions extends ClientOptionsBase {
    * scan leases and renew them. In order to redistribute leases in a timely fashion after a host
    * ceases operating, we recommend a relatively short interval, such as ten seconds. Obviously it
    * should be less than half of the lease length, to prevent accidental expiration.
+   *
+   * If `Leasemanager` is provided then this value will be ignored.
```

Comment:
```suggestion
   * If `Leasemanager` is provided when creating the EventProcessorHost then this value will be ignored.
```

Reply:
The above clarification is needed because LeaseManager is not part of the options